db/migrate/20160505005253_add_send_count_to_invitations.rb

```diff
@@ -0,0 +1,5 @@
+class AddSendCountToInvitations < ActiveRecord::Migration
+  def change
+    add_column :invitations, :send_count, :integer, null: true, default: 0
```

Comment:
great article!

Seems like the most important index will be `accepted_at`.

```
add_index :invitations, :accepted_at, where: 'accepted_at IS NULL'
add_index :invitations, :send_count, where: 'accepted_at IS NULL'
```

Also worth nothing that we could combine them (the indices into 1) but I don't think there is a lot of benefit to that.